packages/composite-checkout/src/components/radio-button.tsx

```diff
@@ -10,69 +10,6 @@ import PropTypes from 'prop-types';
 import { Theme } from 'src/lib/theme';
 import styled from '../lib/styled';
 
-export default function RadioButton( {
```

Comment:
Because of the way github displays this diff and the `CheckoutSteps` one, it's hard to tell if these components actually changed or if they were just moved around. So that I don't have to go compare them line-by-line, do you know what the differences are?

Reply:
The only changes happen here: https://github.com/Automattic/wp-calypso/pull/46903/files#diff-756c6ffb6eecc8ea890046c286928cd2631d88b9ee52e9c3157da29f17240b47R461

Other than that, it's all moving things around. I think I broke the tests by trying to update the `CheckSteps` function due to a typescript error. @michaeldcain said he would take a look.

Reply:
Thanks! In the future, for PRs like this it can be helpful to move things around in one commit, then do the changes in a separate commit, so that they can be reviewed separately.

Reply:
(If you can't commit because of TypeScript or linting errors while making separate commits, you can do `git commit --no-verify` to make a commit succeed even though you know there are errors, then take care of the errors later.)

Reply:
I was wondering if there was a better way. Thanks for that suggestion. I'll give that a shot for next time.